README.md

```diff
@@ -21,12 +17,32 @@ You can setup the enviroment locally by executing this command:
 
     $ ./bin/local-provision <cloud_credentials_path>
 
-The ```<cloud_credentials_path>``` required by the command indicates a path to a novarc file with the OpenStack credentials that will be used by the Jenkins slave instances to spin up Snappy instances. The scripts copies it to the Jenkins container and the jobs use it to access to the cloud provider API.
+This command creates three kind of containers:
+
+* Jenkins master instance: the actual Jenkins server, the jobs, plugins and configuration options are stored here.
+
+* Jenkins slave instances: the jobs are executed in this containers. We have instances belonging to different distributions so that we can get tests results for each of them.
```

Comment:
I would write it like: We have instances of different distributions... results FROM each of them.
Also I think it might be nice to mention that the slaves have a label with their distribution adjective code name, to make the selection in jobs.